frontend/src/routes/+page.svelte

```diff
@@ -45,27 +45,40 @@
   }
 
   const search = async () => {
-    // Builds the optional query for genres
-    // Example: "?g=Action&g=Comedy&g=Drama"
-
     if (!currentMediaAmount) {
       setViewportToDefault()
     }
 
+    // Builds the optional query for genres
+    // Example: "?g=Action&g=Comedy&g=Drama"
     let query = ""
+    let userInput = input.replace(/[^\w\d\s\&]/g, " ")
```

Comment:
We should probably add a comment with something like:
```suggestion
    // Sanitizes input by removing any character that is not a letter, number, whitespace, or an ampersand
    // and replaces them with a space
    let userInput = input.replace(/[^\w\d\s\&]/g, " ")
```